scripts/us_nces/demographics/private_school/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "import_specifications": [
+    {
+      "import_name": "NCES_PrivateSchoolStats",
+      "curator_emails": ["support@datacommons.org"],
+      "provenance_url": "https://nces.ed.gov/ccd/elsi/tableGenerator.aspx",
+      "provenance_description": "US nces school data for private",
+      "scripts": ["run.sh" ,"process.py"],
+      "source_files": ["gcs_output/input_files/*"],
+      "import_inputs": [
+        {
+          "template_mcf": "gcs_output/output_files/us_nces_demographics_private_school.tmcf",
+          "cleaned_csv": "gcs_output/output_files/us_nces_demographics_private_school.csv"
+                 },
+        {
+        "template_mcf": "gcs_output/output_place/us_nces_demographics_private_place.tmcf",
```

Comment:
Should this file with new schools be added to the import NCES_PrivateSchool? 
Pls add some instructon to readme that describes steps when new schools are generated.